### PR TITLE
removed csrf

### PR DESCRIPTION
**Pull request by facchettos**
### Which issue this PR addresses:
https://issues.redhat.com/browse/ARO-3246
https://issues.redhat.com/browse/ARO-1730

Fixes

### What this PR does / why we need it:
removes csrf protection as the library is deprecated and we are not vulnerable to this kind of attack in any case as the portal is read only as of today. 

### Test plan for issue:
should work with existing tests

### CODE REVIEW

File: portal/v2/src/SSHModal.tsx
Comment (facchettos) [inline portal/v2/src/SSHModal.tsx:1]: for some reason we cannot remove the mutablerefobject from this (at least I couldn't with my limited knownledge of the portal frontend). Open to suggestions here

File: pkg/portal/security_test.go [hunk 1]
```diff
@@ -291,7 +290,6 @@ func TestSecurity(t *testing.T) {
 					t.Fatal(err)
 				}
 
-				err = addCSRF(req)
 				if err != nil {
 					t.Fatal(err)
```
Comment (cadenmarchese) [inline pkg/portal/security_test.go:294]: Same with this one.

**Review by ellis-johnson — APPROVED** [review]
LGTM! Tested locally and it all seems to function normally. Just those two error checks that need to be removed that were already pointed out

File: pkg/portal/http_test.go [hunk 2]
```diff
@@ -81,7 +81,6 @@ func (p *testPortal) Request(method string, path string, authenticated bool, ele
 		return nil, err
 	}
 
-	err = addCSRF(req)
 	if err != nil {
 		return nil, err
 	}
```
[the remark this answers is not in the document]
Comment (facchettos) [reply pkg/portal/http_test.go:86]: Damn it you're right I should be more careful and check what the result is after my vim macros 😭 will fix it once I'm back from Pto

**Pull request closed without merging**